iscservice/demo/README.rst

```diff
@@ -0,0 +1,91 @@
+Preparing computations library
+==============================
+
+Computations from an external library cannot be linked directly with
+a Motr instance. The library is supposed to have an entry function named
+``void motr_lib_init(void)``. All the computations in the library must
+have the following signature::
+
+  int comp(struct m0_buf *args, struct m0_buf *out,
+           struct m0_isc_comp_private *comp_data, int *rc)
+
+See demo/libdemo.c for examples.
+
+Loading the library
+===================
+
+With ``spiel`` command (see spiel/spiel.h and demo/util.h) the library
+can be loaded with any running Motr instance. A helper function
+``m0_isc_lib_register`` takes the library path which is (IMPORTANT!)
+expected to be the same across all the nodes running Motr.
+``m0iscreg`` utility takes the path as an input and loads the library
+into all the remote Motr instances.
+
+On successful loading of the library, the output will look like this::
+
+  $ m0iscreg -e 192.168.180.171@tcp:12345:4:1 \
+             -x 192.168.180.171@tcp:12345:1:1 \
+             -f 0x7200000000000001:0x2c \
+             -p 0x7000000000000001:0x50 $PWD/libdemo.so
+  m0iscreg success
+
+The four options are standard ones to connect to Motr::
+
+  $ m0iscreg -h
+
+  Usage: m0iscreg OPTIONS libpath
+
+     -e <addr>  endpoint address
+     -x <addr>  ha-agent (hax) endpoint address
+     -f <fid>   process fid
+     -p <fid>   profile fid
+
+The values for them can be taken from the output of ``hctl status``
+command. (We'll refer to them as ``<motr-opts>`` below.)
```

Comment:
maybe we can add more:
`m0iscreg` can be used to register other ISC libs without modification, or with slight modification according to different needs.

Reply:
Ok, added.